requirements-test.txt

```diff
@@ -1,27 +1,27 @@
 boto3==1.16.28
 botocore==1.19.28
-certifi==2020.11.8
+certifi==2020.12.5
 click==8.0.0a1
-fsspec==0.8.4
+fsspec==0.8.5
 jmespath==0.10.0
-marshmallow==3.9.1
+marshmallow==3.10.0
 matplotlib==3.3.3
-numpy==1.19.4
-pandas==1.1.4
+numpy==1.20.0
```

Comment:
This is causing the 3.6 test to fail FYI.

Reply:
we were thinking of removing 3.6 req no ?

Reply:
```suggestion
numpy>=1.19.4
```

Reply:
> we were thinking of removing 3.6 req no ?

3.6 hasn't reached EOL yet - so I'd expect people still have it running. Also, not reason to drop Python support because of numpy dependeny